queries.sql

```diff
@@ -1,3 +1,64 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name = '%mon';
+SELECT * FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT * FROM animals WHERE neutered='True' AND escape_attempts<3; 
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon','Pikachu'); 
+SELECT name, escape_attempts FROM animals WHERE weight_kg>10.5;
+SELECT * FROM animals WHERE neutered='True'; 
+SELECT * FROM animals WHERE name NOT IN ('Gabumon');
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+/* Day 2 */
+
+/* Transaction 1 */
+
```

Comment:
- [ ] It would be better to begin or start a transaction with the `BEGIN;` keyword and end a transaction with the `COMMIT` or `ROLLBACK` statement. We can implement it in our transaction 1 and 2 since our queries are to be in a transaction as in the guidelines
![Screenshot from 2022-12-13 11-31-45](https://user-images.githubusercontent.com/46388113/207307037-e1b46143-2dc9-44fc-a990-e521ec5849bd.png)

Reply:
Ok. I am doing that immediately.

Reply:
I made the necessary changes.